src/bin/pg_autoctl/pgctl.c

```diff
@@ -671,18 +680,8 @@ prepare_guc_settings_from_pgsetup(const char *configFilePath,
 		}
 	}
 
-	/*
-	 * Only bother to apply settings on Linux, macOS and Windows are considered
-	 * development environment and we decide to refrain from tuning a dedicated
-	 * Postgres system on those targets.
-	 *
-	 * Also disable Postgres tuning when running the unit test suite.
-	 */
-#if defined(__linux__)
```

Comment:
I don't see this linux check anywhere now. Is that on purpose?

Reply:
Good point. Unfortunately there's no way to comment code that's removed, and I forgot to add a note about that in the PR itself. Yes it's on purpose. You reported that your laptop was fine with running more than 4 Postgres nodes with 1/4th of the RAM in shared_buffers for each, for one thing, and for another thing, I don't think we can decide that only Linux is production, come to think of it.

Also makes it easier to test what's happening for others, because it's happening the same on all platforms now.